### PR TITLE
Added Changelog page for components

### PR DESCRIPTION
**Pull request by zdmytriv**
## what

Added Changelog page for components

## why
* Added Changelog page for components

### CODE REVIEW

File: docusaurus.config.js [hunk 1]
```diff
@@ -37,6 +37,26 @@ const config = {
         sidebarPath: require.resolve('./sidebars-components.js')
       },
     ],
+    [
+      require.resolve('./src/plugins/changelog/index.js'),
+      {
+        blogTitle: 'Components Changelog',
+        blogDescription: 'Keep yourself up-to-date about new features in every release',
```
Comment (osterman) [inline docusaurus.config.js:44]:
```suggestion
        blogDescription: 'Stay informed about new features and breaking changes in every release',
```

File: docusaurus.config.js [hunk 2]
```diff
@@ -37,6 +37,26 @@ const config = {
...
+        blogSidebarCount: 'ALL',
+        blogSidebarTitle: 'Changelog',
+        routeBasePath: '/components/changelog',
+        showReadingTime: false,
+        postsPerPage: 20,
+        archiveBasePath: null,
+        authorsMapPath: 'authors.json',
+        feedOptions: {
+          type: 'all',
+          title: 'Cloud Posse components changelog',
```
Comment (osterman) [inline docusaurus.config.js:54]:
```suggestion
          title: 'Cloud Posse Components Changelog',
```

File: src/plugins/changelog/theme/ChangelogItem/Header/index.tsx [hunk 3]
```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
```
Comment (osterman) [inline src/plugins/changelog/theme/ChangelogItem/Header/index.tsx:5]: We need to correct this. Correct this language by adding the `LICENSE` to the `ChangelogItem/` folder. See https://github.com/grafana/grafana/blob/main/NOTICE.md for example.

Comment (osterman) [reply src/plugins/changelog/theme/ChangelogItem/Header/index.tsx:5]: Update this file and related files.

File: .github/workflows/website-deploy-preview.yml [hunk 4]
```diff
@@ -61,7 +61,8 @@ jobs:
         with:
           fetch-depth: 1
           repository: cloudposse/terraform-aws-components
-          ref: main
+          # ref: main
+          ref: upgrade-guide-workflow
```
Comment (zdmytriv) [inline .github/workflows/website-deploy-preview.yml:65]: this should be fixed before merge

**Review by milldr — APPROVED** [review]

**Pull request merged**